terraform/environments/ppd/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+terraform {
+  backend "s3" {
+    bucket = "terraformer-remote-states-use1"
+    key    = "apps/magine/ppd.tfstate"
+    region = "us-east-1"
+  }
+}
+
+module "magine" {
+  source = "../../modules/magine"
+
+  region        = "us-east-1"
+  environment   = "ppd"
+  assets_bucket = "maisonette-ppd"
+  cms_bucket    = "maisonette-cms-ppd"
```

Comment:
Legacy buckets? Names could be more descriptive if they are new buckets.

Reply:
Well, the `cms` one is better, but `maisonette-ppd` is quite generic

Reply:
maisonette-ppd is the shared assets bucket for the env. Agree it could be better but this isn't going to change soon.